_posts/blog/2017-05-31-conversational-interfaces.md

```diff
@@ -0,0 +1,54 @@
+
+## Be Concise 
+
+Unlike a GUI, where users can read and decide on options at their own pace, once a verbal response is given, users must rely on their memory of the spoken response to understand the next step — be concise and structure your requests for easy recall. Additionally, skimming a paragraph isn’t an option with a voice interface, which will frustrate a user who wants to advance quickly — to prevent this, be concise with your responses and remember foremost to answer the question the user asked. The whole experience must be easier than the alternative, or your users won’t be around for long.
+ 
+## Prevent Frustrating Moments
+
+Sometimes there are tough moments when users either didn’t say anything, or their response was misunderstood. Users have less tolerance for error in voice interfaces, because the root cause can be less obvious (you _know_ you made a mistake in a GUI when you fat-fingered something!) — how do you recover gracefully from these situations? 
+ 
+Acknowledge the ambiguity as soon as possible, since it will be more difficult to unravel later. Attempt to handle the correction using normal conversation. This can be accomplished by building a list of response prompts and randomizing them. Simply restating the exact question will come off as rude and impatient. Additionally, consider breaking the question into parts, or helping the user understand why you need more information. Understanding their progress through the experience will make them feel more comfortable answering the questions.
+ 
+Finally, as a general rule, if you have unsuccessfully asked the question three times (because presumably you must have the information to continue), exit the experience gracefully and hope it goes more smoothly next time.
+
+
+## Design a Great Experience
+
+After listening to several experts in the conversation UI field last week, I felt like all the messages boiled down to one maxim: 
+ 
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**Always imagine the whole experience you are creating as a conversation between two people.**
+ 
+Viewed through the lens of social conventions, all these other tips are fairly obvious when you think about how you would like a real person to respond. 
+ 
+That being said, as I listened to all of these special cases, the engineer in me started to panic while mentally tabulating the growing lines of code required. This sounds like a LOT of extra work! Indeed it is; but, to ensure my users have a great experience, I’m now motivated to develop and design for that extra effort.
+ 
+I hope you’ve found this primer helpful. If you’d like more detailed information from the experts, here are some great sessions from Google I/O 2017 on developing conversational user interfaces:
+
+[Finding the Right Voice Interactions for your App](https://www.youtube.com/watch?v=0PmWruLLUoE&list=PLOU2XLYxmsIKC8eODk_RNCWv3fBcLvMMy&index=78)
```

Comment:
Let's make these links open new tabs so readers don't lose the article page. Jekyll supports [Kramdown attribute lists](https://kramdown.gettalong.org/syntax.html#inline-attribute-lists), so you just need to add `{:target='_blank'}` after each link, e.g.:

```markdown
[Finding the Right Voice Interactions for your App](https://www.youtube.com/watch?v=0PmWruLLUoE&list=PLOU2XLYxmsIKC8eODk_RNCWv3fBcLvMMy&index=78){:target='_blank'}
```